Rocket.Chat/Managers/Socket/SocketManager.swift

```diff
@@ -164,36 +164,36 @@ extension SocketManager {
                 return
             }
 
-            infoClient.fetchInfo(completion: {
-                SubscriptionManager.updateSubscriptions(auth) {
-                    AuthSettingsManager.updatePublicSettings(auth)
-
-                    UserManager.userDataChanges()
-                    UserManager.changes()
-                    SubscriptionManager.changes(auth)
-                    SubscriptionManager.subscribeRoomChanges()
-                    SubscriptionManager.subscribeInAppNotifications()
-                    PermissionManager.changes()
-                    infoClient.fetchPermissions()
-                    CustomEmojiManager.sync()
-
-                    commandsClient.fetchCommands()
-
-                    if let userIdentifier = auth.userId {
-                        PushManager.updateUser(userIdentifier)
-                    }
+            infoClient.fetchInfo()
```

Comment:
I'm not sure why these changes are being showed here 🤔 have you modified these files?

Reply:
Oh, I merged the `bottom-spacing` branch and then reverted the changes. Maybe that is what's the issues.